storage_service/locations/models/package.py

```diff
@@ -783,7 +785,10 @@ def check_fixity(self, delete_after=True):
         if self.is_compressed:
             # bagit can't deal with compressed files, so extract before
             # starting the fixity check.
-            path, temp_dir = self.extract_file()
+            try:
+                 path, temp_dir = self.extract_file()
+            except StorageException:
+                 return (False, [], 'Error extracting file')
```

Comment:
I think this should be `None` not `False` - the docstring says `False` is for a verification failure, and `None` is for a scan that couldn't start.